### ISSUE TITLE
validate ORCID iDs

### ISSUE THREAD

**Issue by atmodatcode** [AtMoDat/atmodat_data_checker#122]
For future ATMODAT versions, we could consider checking if users correctly specified their ORCIDs, especially in the property creator_id. 
This is how it should work.

import re
pattern = re.compile(r"^(https://orcid.org/[0-9]{4}-[0-9]{4}-[0-9]{4}-[0-9]{3}[0-9|X])+$")
orcid = "https://orcid.org/0000-0001-7057-194X"
bool(re.match(pattern, orcid))  # False if ORCID syntax is wrongly specified

**Comment by jkretz** [comment]
Yes, we could relatively easily implement that. The only issue I see is that we need to know in advance in which global attributes an ORCID could be present. At the moment, we cannot run such a check over all global attributes.

**Comment by jkretz** [comment]
I set up the functionality for this check in the orcid_check branch. As long as it is not in the ATMODAT standard, I am not sure if we should merge it. To activate those checks, they have to be added to the  [atmodat_standard_checks.yml](https://github.com/AtMoDat/AtMoDat_CVs/blob/master/atmodat_standard_checks.yml).
